spec.md

```diff
@@ -112,6 +113,43 @@ This header allows brokers to reject requests from marketplaces for versions the
 The marketplace MUST authenticate with the service broker using HTTP
 basic authentication (the `Authorization:` header) on every request. The broker is responsible for validating the username and password and returning a `401 Unauthorized` message if credentials are invalid. It is RECOMMENDED that brokers support secure communication from platform marketplaces over TLS.
 
+## Originating Identity
+
+Often a service broker will need to know the identity of the user that
+initiated the request from the platform. For example, this might be needed
+for auditing or authorization purposes. In order to facilitate this, the
+platform will need to provide this identification information to the broker
+on each request. If the platform wishes to provide this information then
+it MUST include an additional HTTP header on all requests it sends to the
+broker, except the request to retrieve the catalog since that is not
+associated with a platform's user. The format of the header MUST be:
```

Comment:
What if I reword this:
```
If the platform wishes to provide this information then it MUST include an additional HTTP
header on all requests it sends to the broker, except the request to retrieve the catalog
since that is not associated with a platform's user.
```
to be:
```
Platforms MAY support this feature. If they do, they MUST adhere to the following:
- for any OSB API request that is the result of an action taken by
  a platform's user, there MUST be an associated OriginatingIdentity header on that
  HTTP request.
- any OSB API request that is not associated with an action from a platform's user,
  such as the platform querying the marketplace, MAY exclude the header from that
  HTTP request.
- if present on a request, the OriginatingIdentity header MUST contain the identify
  information for the platform's user that took the action to cause the request to be sent.
```